pkg/server/backuprestoreserver.go

```diff
@@ -72,12 +74,15 @@ func NewBackupRestoreServer(logger *logrus.Logger, config *BackupRestoreComponen
 		// Ideally this case should not occur, since this check is done at the config validaitions.
 		return nil, err
 	}
+	backoffExponentialConfig := brtypes.NewExponentialBackOff()
+
 	return &BackupRestoreServer{
 		logger:                  serverLogger,
 		config:                  config,
 		ownerChecker:            ownerChecker,
 		etcdProcessKiller:       etcdProcessKiller,
 		defragmentationSchedule: defragmentationSchedule,
+		backoff:                 backoffExponentialConfig,
```

Comment:
Should not we name it like `backoffConfig` ?

Reply:
I haven't made the [ExponentialBackoff struct](https://github.com/gardener/etcd-backup-restore/pull/411/files#diff-ff42c8f8fcbf5c49ef29bde3eced688de7d922868435ce048b6d40c049c6ebd8R35)  configurable as I thought in backup-restore we are already having too many configurable flags. 
so, Should I change the name from `backoff` to `backoffConfig`? Isn't the `backoffConfig` is bit misleading ? WDYT?

Reply:
I think, `backoffConfig` is fine. Config means `configuration` here.  `backoff` sounds like a command

Reply:
ok, I will make the necessary  changes.

Reply:
Addressed. PTAL